components/TextFields/src/private/MDCTextInputAllCharactersCounter.m

```diff
@@ -14,7 +14,9 @@
  limitations under the License.
  */
 
-#import "MaterialTextFields.h"
+#import "MDCTextInputCharacterCounter.h"
+
+#import "MDCTextInput.h"
```

Comment:
Nit: I believe `MDCTextInput.h` should be ordered before `MDCTextInputCharacterCounter.h` (NULL < 'C').  Also please remove the extra newline.

Reply:
I put the `MDCTextInputCharacterCounter.h` first, and separated with an empty line, because it is the main declaration import. This is a common practice, that is also used around your code base. Should I still make changes suggested by you?

Reply:
Your formatting was correct. I didn't realize that the header declared this class as well.  Sorry!